src/legacy/core_plugins/data/public/query/query_bar/components/query_bar_input.tsx

```diff
@@ -196,10 +196,8 @@ export class QueryBarInputUI extends Component<Props, State> {
   };
 
   private onQueryStringChange = (value: string) => {
-    const hasValue = Boolean(value.trim());
-
     this.setState({
-      isSuggestionsVisible: hasValue,
+      isSuggestionsVisible: true,
```

Comment:
We originally changed this when the saved query UI was in the autocomplete dropdown so that you could access those controls even if the query was blank. We removed the UI but we kept this change because we liked how you could see the autocomplete without having to type anything.

Reply:
That's great! I'd love this behavior :)